### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by eudondian**
-  a new game with the name my cool game was created by using the API.
-  Implement the "Refresh" button (receiving data from the API and parsing the JSON).
-  Implement the form "Submit" button (sending data to the API).
-  Arrow functions instead of the function keyword were used.
-  Async and await JavaScript features were used to consume the API.
-  No styling yet

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  ).then((res) => res.json().then((data) => {
+    uniqueIdentifier = data.result;
+  }));
+};
+gamer();
+
+submit.addEventListener('click', async () => {
+  const newScore = new Scores(name.value, score.value);
+  await fetch(
+    `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${uniqueIdentifier}/scores/`,
```
Comment (FrederickMih) [inline src/index.js]: - [ ] When the submit button is clicked, data is not successively posted to the Leaderboard API. It seems you didn't generate a unique id for POST and GET actions. Please kindly fix this issue.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @eudondian, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



# To Highlights

- GitHub workflow has been followed :heavy_check_mark: 
- Linters checks are OK :heavy_check_mark: 
- PR has a title and summary :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

##### Kindly include all the headers from [this template](https://github.com/microverseinc/readme-template) in your README file and also fix the License link

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (eudondian) [reply src/index.js]: https://www.loom.com/share/8f591595b42f44aeae57bd4d3fbe7d76

Comment (FrederickMih) [reply src/index.js]: Hi @eudondian, kindly refresh the page after posting data.

**Review by topeogunleye — APPROVED** [review]
Hi @eudondian, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

dwayne-the-rock

## (Highlights) Good Points: 💪 
* Good commit messages. :heavy_check_mark:
* No linter errors. :heavy_check_mark:
* Followed Gitflow :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
>- [ ] Next time, and for future Pull Requests, please kindly add a more descriptive PR title.
> See a demo below:
pull_request_desc

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**